setup.py

```diff
@@ -32,7 +32,7 @@
     # Versions should comply with PEP440.  For a discussion on single-sourcing
     # the version across setup.py and the project code, see
     # https://packaging.python.org/en/latest/single_source_version.html
-    version='0.1',
+    version='0.5',
```

Comment:
I don't really have a strong feeling.  I'm OK with jumping from 0.1 (the old stuff) to 0.5 with the new merge of Stephanie's updates and then going from there. Hey - ESRI jumped from ArcView 3.3 to ArcGIS 8.something.  I always wondered about that...